src/argus_htmx/templates/htmx_base.html

```diff
@@ -5,7 +5,10 @@
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     {% block title %}<title>Argus Server: {{ page_title }}</title>{% endblock %}
-
+    <!-- HTMX -->
+    <script src="{% static "htmx-2.0.2.min.js" %}"></script>
```

Comment:
You know what would be cool, a template filter that checks the current static directory for htmx files and picks the latest. something with `pathlib.Path.glob` together with  a `version_info` namedtuple or so. slap a `@functools.cache` on it for performance

Reply:
Thanks to having to take the type of storage in use into account (filenames can be rewritten) this is a big enough change that it deserves its on issue.

Reply:
We don't neccesarily need to keep old versions I think, but it makes the aforementioned Make command easier if the template has some htmx version discovery. The make command can then delete old versions, add the new one, and Django automatically picks up on it